locale/locale.bzl

```diff
@@ -16,8 +16,10 @@ def _impl(ctx):
         ],
         command = """
         set -o pipefail -o errexit -o nounset
+        ar -x "$1" data.tar.xz
```

Comment:
could this just happen after `tmp=$(mktemp -d)` and use the `--output` option in `ar`

Reply:
Modify it according to the comments.